test/aave/CloseStEth.test.ts

```diff
@@ -384,25 +384,31 @@ describe(`Strategy | AAVE | Close Position`, async () => {
           .then((amount: ethers.BigNumberish) => amountFromWei(new BigNumber(amount.toString())))
 
         const positionAfterOpen = new Position(
-          { amount: new BigNumber(beforeCloseUserAccountData.totalDebtETH.toString()) },
+          {
+            amount: new BigNumber(beforeCloseUserAccountData.totalDebtETH.toString()),
+            symbol: tokens.ETH,
+          },
           {
             amount: new BigNumber(beforeCloseUserStEthReserveData.currentATokenBalance.toString()),
+            symbol: tokens.STETH,
           },
           aaveStEthPriceInEth,
           openStrategy.simulation.position.category,
         )
 
-        closeStrategy = await strategies.aave.closeStEth(
+        closePositionMutation = await strategies.aave.close(
```

Comment:
What's with the naming `closePositionmutation`? Can't we just name it `closePosition` and that's it ?